Cargo.toml

```diff
@@ -13,9 +13,22 @@ Simple and fast implementation of language tag normalization and validation
 """
 edition = "2018"
 
+[features]
+default = []
+serialize = ["serde"]
```

Comment:
I would maybe omit completely the "serialize" feature in favor of just depending on the "sled" dependency being present or not (with just `#[cfg(feature = "serde")]` in the code). This would make usages simpler. Because of trait impls constraints, adding `Serialize` and `Deserialize` implementations should not break any user of this crate because the trait can only be implemented here in oxilangtag or in Serde (where it it isn't...).

Reply:
I do not believe I understand what you mean with this. What do you mean by depending on `sled`? Isn't that the KV store database? 

This should not break compatibility, since users can not implement traits onto foreign types.

Reply:
Sorry, I mixed up "sled" and "serde". Sorry for the confusion.